pkg/get/get_test.go

```diff
@@ -7701,13 +7701,13 @@ func Test_DownloadKeploy(t *testing.T) {
 			os:      "ming",
 			arch:    arch64bit,
 			version: toolVersion,
-			url:     "https://github.com/keploy/keploy/releases/download/v2.3.0/keploy_windows_amd64.zip",
+			url:     "https://github.com/keploy/keploy/releases/download/v2.3.0/keploy_windows_amd64.tar.gz",
```

Comment:
Your releases page should really ship a .zip file

Have you thought of changing it?